feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.ts

```diff
@@ -20,29 +20,39 @@ import { ConfiguratorStorefrontUtilsService } from '../../../service/configurato
 export class ConfiguratorAttributeBaseComponent {
   protected configuratorUISettingsConfig = inject(ConfiguratorUISettingsConfig);
   protected translation = inject(TranslationService);
-  protected configuratorDeltaRenderingService = inject(
-    ConfiguratorDeltaRenderingService,
+  protected configuratorAttributePriceChangeService = inject(
+    ConfiguratorAttributePriceChangeService,
     { optional: true }
   );
```

Comment:
It's not obvious, so let's add a code comment here that this service is stateful, so it should be provided in each component's instance separately in their section of the decorator `@Component({ providers: [ConfiguratorAttributePriceChangeService] })`.